### PR TITLE
Syndicate Ninja

### PR DESCRIPTION
**Pull request by yackemflam**
:cl: Yackemflam
add: The syndicate has learned on how to give their agents a true syndicate ninja experience.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -131,6 +131,19 @@
 			new /obj/item/weapon/grenade/plastic/c4 (src) // 1 tc
 			new /obj/item/weapon/card/emag(src) // 6 tc
 			return
+		
+		if("ninja") // 33 tc worth
+			new /obj/item/weapon/katana(src) // Unique , hard to tell how much tc this is worth. 8 tc?
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: It's an esword you can't hide 5-6 tc

Comment (yackemflam) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: It's better than an esword in terms of damage, 40 force

**Comment by PKPenguin321** [comment]
>this PR again

what have you changed since last time to make it not shit? or are you just pushing the same denied PR and hoping the guy that closed it last time wont notice?

**Comment by yackemflam** [comment]
I added in the chameleon projector.
I also used the original katana instead of a weaker variant.
I also removed the mallninja memes since no one liked it.

**Comment by Razharas** [comment]
Its fine

File: code/game/objects/items/weapons/storage/uplink_kits.dm (hunk 1 above)
Comment (jughu) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Its also a emagh

Comment (KorPhaeron) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: why the fuck did github change its format again

Comment (jughu) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Idk

Comment (ShadowDeath6) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: not the ekatana, it doesn't have emag powers

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 2]
```diff
@@ -131,6 +131,19 @@
...
+		
+		if("ninja") // 33 tc worth
+			new /obj/item/weapon/katana(src) // Unique , hard to tell how much tc this is worth. 8 tc?
+			new /obj/item/weapon/implanter/adrenalin(src) // 8 tc
+			new /obj/item/weapon/throwing_star(src) // 5 tc for all 5
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
```
Comment (nullbear) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Someone is really bad at counting.

Comment (yackemflam) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: I made that comment when there was 5 throwing stars and forgot to change it later, whooops

**Comment by KorPhaeron** [comment]
Getting a giant weapon that you can't hide seems sort of undesirable as traitor

**Review by PKPenguin321 — COMMENTED** [review]
apparently i have to make a review if i want to comment on code at all
???

**Pull request merged**